tests/integration/transactions/test_trust_set.py

```diff
@@ -92,3 +96,45 @@ async def test_special_chars_curr_codes(self, client):
             error.exception.args[0],
             "{'currency': 'Invalid currency abcd'}",
         )
+
+    @test_async_and_sync(globals())
+    async def test_deep_freeze_functionality(self, client):
+        issuer_wallet = Wallet.create()
```

Comment:
you also need to fund the `issuer_wallet` into existence. `Wallet.create()` only generates the public-private key pairs for an account.

We need to fund an account with a minimum XRP value for it to be recognised by the XRPL.

Reply:
I also thought this could be the error. @achowdhry-ripple  I think it's worth trying to fund the account to see if the error gets resolved.

Reply:
@achowdhry-ripple 
Here is a concrete example. You'll need to mimic this code: https://github.com/XRPLF/xrpl-py/blob/7af0ca8ba0f8bfb20f090cef8a77e31ad5a9ab6f/tests/integration/transactions/test_clawback.py#L29